controllers/user.js

```diff
@@ -13,21 +13,11 @@ class Users {
     * @constructor
     * @param {object} req - The request object.
     * @param {object} res - The response object.
+    * @param {object} next -The next middleware
     */
-  static getUser(req, res) {
-    User.findById(req.params.userId)
-      .then((user) => {
-        if (!user) {
-          return res.status(404).json({
-            message: 'User Not Found'
-          });
-        }
-        return res.status(200).json({
-          message: 'User Found',
-          user
-        });
-      })
-      .catch(error => res.status(400).send(error));
+  static async getUser(req, res) {
+    const user = await User.findOne({ where: { id: req.params.userId } });
+    return res.json({ message: 'success', user });
```

Comment:
I don't think we should include this `message` key-pair value here. Just a user object will be good.

Also, we don't want to send the whole user object back unless absolutely necessary.

What you are sending back will contain `password` for instance, and we know there's no use for an hashed password.

The user object you should send back should be one you have modified to get only necessary details.

Reply:
OK. Refactoring that now